utils/spdx/src/main/kotlin/SpdxUtils.kt

```diff
@@ -179,3 +180,14 @@ private fun getLicenseTextFile(id: String, dir: File): File? =
     ).mapNotNull { filename ->
         dir.resolve(filename).takeIf { it.isFile }
     }.firstOrNull()
+
+/**
+ * Try to locate the directory which contains the license texts provided by ScanCode. This requires ScanCode to be
+ * installed and the `scancode` binary to be on the path.
+ */
+fun getScanCodeLicenseTextsDir(): File? {
+    val scanCodeDir = getPathFromEnvironment("scancode")?.parentFile?.absoluteFile
+
+    return scanCodeDir?.resolve("src/licensedcode/data/licenses")?.takeIf { it.isDirectory }
+        ?: scanCodeDir?.resolve("../lib/site-packages/licensedcode/data/licenses")?.takeIf { it.isDirectory }
```

Comment:
Maybe in preparation for potentially adding more candidate paths do:

    return listOf(
        "src/licensedcode/data/licenses",
        "../lib/site-packages/licensedcode/data/licenses"
    ).mapNotNull { relativePath ->
        scanCodeDir?.resolve(relativePath)?.takeIf { it.isDirectory }
    }.firstOrNull()